packages/docs/docs/versions/index.md

```diff
@@ -0,0 +1,114 @@
+---
+sidebar_label: Versions
+sidebar_position: 1
+---
+
+# Medplum Version Policy
+
+## Overview
+
+Medplum follows semantic versioning (semver) for all components of our healthcare developer platform. This document outlines our version release strategy, support timeline, and upgrade requirements to help organizations plan their deployment and maintenance schedules effectively.
+
+![Medplum Versions](./medplum-versions.webp)
+
+## Component Versioning
+
+All Medplum components are released in lockstep with the same version number:
+
+- API Server
+- Client SDKs
+- React Component Library
+- Agent
+- Supporting tools and utilities
+
+This means when version X.Y.Z is released, all components are updated to that version simultaneously, ensuring consistent compatibility across the platform.
+
+## Release Schedule
+
+### Major Versions (X.0.0)
+
+- Released annually in Q4
+- Support lifecycle consists of:
+  - 1 year "Active" status with full support and feature updates
+  - 1 year "Maintenance" status with critical updates only
+- Primarily driven by updates to core dependencies
+- Include any necessary breaking changes, concentrated into the annual release
+
+### Minor Versions (X.Y.0)
+
+- Released 2-3 times per year
+- Require server maintenance including potential database migrations
+- Must be deployed sequentially (cannot skip minor versions)
+- Include backwards-compatible feature additions and improvements
+
+### Patch Versions (X.Y.Z)
+
+- Released approximately weekly
+- Include bug fixes and non-breaking feature additions
+- Can be deployed directly without intermediate steps
+
+## Dependency Support
+
+### Scheduled Release Dependencies
+
+#### Node.js
+
+- Supports current "Active" and "Maintenance" LTS Node.js versions
+- All unit and integration tests are run against supported versions
+- Follows Node.js even-numbered LTS release schedule (16.x, 18.x, 20.x, etc.)
+
+#### PostgreSQL
+
+- Supports all PostgreSQL LTS versions that overlap with Medplum "Active" releases
```

Comment:
Linking to https://www.postgresql.org/support/versioning/ and https://nodejs.org/en/about/previous-releases up above for Node would be nice.